localization/autoware_landmark_based_localizer/autoware_lidar_marker_localizer/package.xml

```diff
@@ -31,6 +31,8 @@
   <depend>tf2_ros</depend>
   <depend>tf2_sensor_msgs</depend>
 
+  <exec_depend>lidar_marker_localizer</exec_depend>
```

Comment:
```suggestion
```
`lidar_marker_localizer` does not exist in Autoware.

Reply:
Did you look at [this line](https://github.com/autowarefoundation/autoware_universe/blob/main/localization/autoware_landmark_based_localizer/autoware_lidar_marker_localizer/launch/lidar_marker_localizer.launch.xml#L2) and get it detected?

If so, I'll create a pr to fix [this line](https://github.com/autowarefoundation/autoware_universe/blob/main/localization/autoware_landmark_based_localizer/autoware_lidar_marker_localizer/launch/lidar_marker_localizer.launch.xml#L2).

Reply:
@YamatoAndo Yes, searching for `find-pkg-share` and adding it as `exec_depend`.

Reply:
fixed in https://github.com/autowarefoundation/autoware_universe/pull/10405

Reply:
Thank you. I re-ran the tool. https://github.com/autowarefoundation/autoware_universe/pull/10404/commits/affd791c876b05aca75bbcaf3d59d7491cb00f23